src/integration/pages/homePage/canonicalTests.js

```diff
@@ -0,0 +1,40 @@
+import { runCoreCanonicalTests } from '../../common';
+import runCrossPlatformTests from './crossPlatformTests';
+
+export default service => {
+  runCrossPlatformTests(service);
+  runCoreCanonicalTests();
+  describe.only('Page content', () => {
+    const pageData = window.SIMORGH_DATA.pageData;
```

Comment:
This is how to fix object destructuring lint error.
```suggestion
    const { pageData } = window.SIMORGH_DATA;
```